dolphinscheduler-ui/src/views/login/index.tsx

```diff
@@ -38,15 +41,16 @@ import { useLocalesStore } from '@/store/locales/locales'
 import { useThemeStore } from '@/store/theme/theme'
 import cookies from 'js-cookie'
 import { ssoLoginUrl } from '@/service/modules/login'
+import { OAuth2Provider } from '@/service/modules/login/types'
```

Comment:
Please import type.

Reply:
I have already resolve it, could you please help review it again when you are free @songjianet